examples/next/default.vcl

```diff
@@ -0,0 +1,30 @@
+vcl 4.1;
+
+backend node {
+  .host = "node";
+  .port = "3000";
+}
+
+sub vcl_recv {
+    if (req.http.upgrade ~ "(?i)websocket") {
```

Comment:
WebSockets support should be synchronized (enabled or disabled) with the express example.